### ISSUE TITLE
More detailed example on how to load rdkit-js in various javascript and typescript context

### ISSUE THREAD

**Issue by MichelML** [rdkit/rdkit-js#159]
See context of this issue here https://github.com/rdkit/rdkit-js/discussions/150#discussioncomment-3338647

@heberleh more thoughts welcome here :+1:

**Comment by MichelML** [comment]
https://github.com/rdkit/rdkit-js/issues/124

**Comment by MichelML** [comment]
Aligned with this idea, starter templates for projects in various stacks could be great.

**Comment by Ramb0a** [comment]
Interested in a Sveltekit example? Am currently working on it.

**Comment by MichelML** [comment]
@Ramb0a yes of course, and sorry for the delays! I created an issue for you https://github.com/rdkit/rdkit-js/issues/290

Have you been able to load rdkit-js in your project?

**Comment by duerrsimon** [comment]
@twoXes Did you ever get it to work in Svelte?

**Comment by occamsystems** [comment]
Having just now gotten this working with Angular, I can say that by far the biggest (apparently) missing piece of documentation was the need to add the wasm blob and RDKit_minimal.js to the angular.json configuration in order for initRDKitModule() to be loaded.

It took me a couple hours to figure this out, and there is a decent chance that many developers would takes days to get there if they did it at all, since this is not the type of operation that is required for "native" Angular libraries.

I think this information could very easily be added near the top of the page on https://angular.rdkitjs.com/ to prevent such issues.

It may be that other frameworks have a similar missing configuration step that is tripping them up, since, apart from this, everything felt very straightforward using npm install.
